code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -356,18 +368,35 @@
 			to_chat(user, "[G.tech_fluff_string].")
 			G.speak_emote = list("states")
 		if("bio")
-			G.name_color = picked_color
-			G.icon = 'icons/mob/mob.dmi'
+			color = pick(bio_list) //technically not colors, just using the same flowers as tech currerntly
+			G.name_color = tech_list[color]
 			picked_name = pick("brood", "hive", "nest")
 			to_chat(user, "[G.bio_fluff_string].")
-			G.name = "[picked_name] swarm"
-			G.color = picked_color
-			G.real_name = "[picked_name] swarm"
-			G.icon_living = "headcrab"
-			G.icon_state = "headcrab"
+
+			G.name = "[color] [picked_name]"
+			G.real_name = "[color] [picked_name]"
+			G.icon_living = "[theme][color]"
+			G.icon_state = "[theme][color]"
+			G.icon_dead = "[theme][color]"
+
+			to_chat(user, "[G.bio_fluff_string].")
 			G.attacktext = "swarms"
 			G.speak_emote = list("chitters")
 
+
+
+//			G.name_color = picked_color
+//			G.icon = 'icons/mob/mob.dmi'
+//			picked_name = pick("brood", "hive", "nest")
+//			to_chat(user, "[G.bio_fluff_string].")
+//			G.name = "[picked_name] swarm"
+//			G.color = picked_color
+//			G.real_name = "[picked_name] swarm"
+//			G.icon_living = "headcrab"
+//			G.icon_state = "headcrab"
+//			G.attacktext = "swarms"
+//			G.speak_emote = list("chitters")
```

Comment:
don't comment out. Remove, please.

Reply:
Oh, oops, that was meant to be deleted, my appoligies, I will fix it when I get the sprites finished.